test/properties.py

```diff
@@ -0,0 +1,281 @@
+        "bedrooms_amount": 3,
+        "contract_type": "rent",
+        "currency_code": "EUR",
+        "floors_amount": 1,
+        "garage_size": 2,
+        "price": 120000,
+        "terrain_height": 5,
+        "terrain_width": 7
+    }
+
+    def setUp(self) -> None:
+        connect("fast-home-test", host="mongomock://localhost")
+        new_user = User(**self._register_user).save()
+        self._userId = str(new_user.id)
+        new_prop = PropertyDoc(**self._register_prop, owner=self._userId).save()
+        self._propId = str(new_prop.id)
+
+    def tearDown(self) -> None:
+        disconnect()
+
+    def test_successful_delete(self):
+        result = delete_prop({
+            "id": self._propId
+        }, self._userId)
+
+        self.assertEqual(result, ControllerStatus.SUCCESS)
+        self.assertRaises(DoesNotExist, lambda: PropertyDoc.objects.get(id=self._propId))
+
+    def test_nonexistent_property(self):
+        test_id = "60999999a651cad33ea0510f"
+        result = delete_prop({
+            "id": test_id
+        }, self._userId)
+
+        self.assertEqual(result, ControllerStatus.DOES_NOT_EXISTS)
+        self.assertRaises(DoesNotExist, lambda: PropertyDoc.objects.get(id=test_id))
+
+    def test_unauthorized_delete(self):
+        test_id = "60999999a651cad33ea0510f"
+        prop_doc = PropertyDoc.objects.get(id=self._propId)
+        result = delete_prop({
+            "id": self._propId
+        }, test_id)
+
+        self.assertEqual(result, ControllerStatus.UNAUTHORIZED)
+        self.assertNotEqual(str(prop_doc["owner"].id), test_id)
+
+
+class GetPropertyDataTest(TestCase):
+    _email = "test@example.net"
+    _password = "testpass"
+    _userId = ""
+    _propId = ""
+    _register_user = {
+        "username": "testuser",
+        "passwd_hash": generate_password_hash(_password),
+        "email": _email,
+        "profile": {
+            "contact_email": _email
+        }
+    }
+    _register_prop = {
+        "address": "1234 Test Street",
+        "bathrooms_amount": 1.5,
+        "bedrooms_amount": 3,
+        "contract_type": "rent",
+        "currency_code": "EUR",
+        "floors_amount": 1,
+        "garage_size": 2,
+        "price": 120000,
+        "terrain_height": 5,
+        "terrain_width": 7
+    }
+
+    def setUp(self) -> None:
+        connect("fast-home-test", host="mongomock://localhost")
+        new_user = User(**self._register_user).save()
+        self._userId = str(new_user.id)
+        new_prop = PropertyDoc(**self._register_prop, owner=self._userId).save()
+        self._propId = str(new_prop.id)
+
+    def tearDown(self) -> None:
+        disconnect()
+
+    def test_successful_specific_property(self):
+        result = get_property_data(self._propId)
+        obj_address = PropertyDoc.objects.get(id=self._propId).address
+
+        self.assertEqual(result[0], ControllerStatus.SUCCESS)
+        self.assertEqual(result[1].address, obj_address)
```

Comment:
Necesitas verificar con la dirección original, no con otra consulta a la base de datos
```suggestion
        self.assertEqual(result[1].address, self._register_prop["address"])
```
No olvídes borrar la variable obj_address después